openapi.yml

```
@@ -0,0 +1,302 @@
+openapi: 3.1.1
+info:
+  title: Codex API
+  description: |
+    Codex API
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)